Dockerfile

```diff
@@ -19,11 +19,11 @@ RUN apt-get -y install nodejs
 
 # Python 2
 RUN apt-get install -y python-pip
-RUN pip2 install --upgrade setuptools requests requests[security]
+RUN pip2 install --upgrade setuptools
 
 # Python 3
 RUN apt-get install -y python3 python3-pip
-RUN pip3 install --upgrade six setuptools aiohttp wheel requests pyopenssl tox twine
```

Comment:
Did you check that it still allows to run py2 & py3 tests without a problem? I copied these deps from our `.travis.yml` script but didn't have time to check whether there are some unneeded ones...

Reply:
@frosty00 Nah, it doesn't work. I've re-built the container image with those installs deleted, and it fails to execute even `npm run build`:

```
sh: 1: tox: not found
```

So please make sure that we have all the required dependencies installed to run those commands:

```
npm run build
```
```
node run-tests --js
```
```
node run-tests --python2
```
```
node run-tests --python3
```
```
node run-tests --php
```

Reply:
...or, if you don't have the time to check it, we can simply keep those installs in the Dockerfile.